backend/src/main/resources/schema.graphqls

```diff
@@ -12,11 +12,13 @@ enum ResultValue {
 
 enum UserPermission {
   READ_PATIENT_LIST
+  READ_PATIENT_INCL_DEL_LIST
+  SEARCH_PATIENTS
   READ_RESULT_LIST
   EDIT_PATIENT
+  DELETE_PATIENT
   EDIT_FACILITY
   EDIT_ORGANIZATION
-  SEARCH_PATIENTS
```

Comment:
Reordering is to keep in sync with java code.

Reply:
Thanks for the explanatory comment—that makes sense!